esp/esp/program/models/__init__.py

```diff
@@ -281,6 +281,7 @@ class Program(models.Model, CustomFormsLinkModel):
     class Meta:
         app_label = 'program'
         db_table = 'program_program'
+        ordering = ['-id','name']
```

Comment:
The idea was to sort 'chronologically' and also by name. I can remove name if it does not seem to be useful.